src/vos/vos_aggregate.c

```diff
@@ -1461,6 +1533,7 @@ flush_merge_window(daos_handle_t ih, struct agg_merge_window *mw,
 	if (!need_flush(mw, last))
 		return 0;
 
+	D_INFO("PREPARE SEGMENTS\n");
```

Comment:
Should it be removed? otherwise, it could generate too many useless logs when D_INFO enabled.

Reply:
ah crap